src/sentry/static/sentry/app/components/projects/missingProjectMembership.tsx

```diff
@@ -1,27 +1,34 @@
 import React from 'react';
 import styled from '@emotion/styled';
 
-import {addErrorMessage} from 'app/actionCreators/indicator';
+import {addErrorMessage, addSuccessMessage} from 'app/actionCreators/indicator';
 import {joinTeam} from 'app/actionCreators/teams';
 import {Client} from 'app/api';
-import Well from 'app/components/well';
+import Button from 'app/components/button';
+import {Panel} from 'app/components/panels';
+import Tooltip from 'app/components/tooltip';
 import {IconFlag} from 'app/icons';
 import {t} from 'app/locale';
+import TeamStore from 'app/stores/teamStore';
 import space from 'app/styles/space';
-import {Organization, Project, Team} from 'app/types';
+import {Organization, Project} from 'app/types';
 import withApi from 'app/utils/withApi';
 import EmptyMessage from 'app/views/settings/components/emptyMessage';
+import Form from 'app/views/settings/components/forms/form';
+import SelectField from 'app/views/settings/components/forms/selectField';
 
 type Props = {
   api: Client;
   organization: Organization;
   projectId?: string;
+  groupId: string;
```

Comment:
Is this component used where `groupId` is not available?

Reply:
The `groupId` is available in the params from `groupDetails.tsx`, but the `shortId` isn't available.

Reply:
@mikellykels It looks like it's also used here where there is no issue context https://github.com/getsentry/sentry/blob/7d36ffa98147b0380dea1a7a203487384ea3d5ac/src/sentry/static/sentry/app/views/projects/projectContext.jsx#L248-L251

Reply:
oops I missed this comment - I see what you mean now and it does look like there's not going to be issue context for a couple of different cases. I also found that one case doesn't actually use the `MissingProjectMembership` component when it should, and is using the default error https://github.com/getsentry/sentry/blob/7d36ffa98147b0380dea1a7a203487384ea3d5ac/src/sentry/static/sentry/app/views/projects/projectContext.jsx#L253-L254. 

Since there are these other cases where the issue ID isn't available, do you recommend just removing the `groupId`?

Reply:
@mikellykels Yeah, I would make the error message more generic since we use it in other places. Also, just showing the issue id is not helpful for the user anyway, they won't be able to do anything with it.